test/ctia/http/routes/graphql/investigation_test.clj

```diff
@@ -0,0 +1,87 @@
+(ns ctia.http.routes.graphql.investigation-test
+  (:require [clj-momo.test-helpers.core :as mth]
+            [clojure.test :refer [deftest is join-fixtures testing use-fixtures]]
+            [ctia.entity.investigation :as inv]
+            [ctia.entity.investigation.examples :refer [investigation-maximal]]
+            [ctia.test-helpers.auth :refer [all-capabilities]]
+            [ctia.test-helpers.core :as helpers]
+            [ctia.test-helpers.fake-whoami-service :as whoami-helpers]
+            [ctia.test-helpers.fixtures :as fixt]
+            [ctia.test-helpers.graphql :as gh]
+            [ctia.test-helpers.store :refer [test-for-each-store]]))
+
+(use-fixtures :once (join-fixtures [mth/fixture-schema-validation
+                                    helpers/fixture-properties:clean
+                                    whoami-helpers/fixture-server]))
+
+(use-fixtures :each whoami-helpers/fixture-reset-state)
+
+(def ownership-data-fixture
+  {:owner "foouser"
+   :groups ["foogroup"]})
+
+(def investigation-1 (fixt/randomize investigation-maximal))
+(def investigation-2 (assoc
+                      (fixt/randomize investigation-maximal)
+                      :source "ngfw"))
+
+(defn prepare-result
+  [investigation]
+  (dissoc investigation
+          :actions
```

Comment:
The `actions` is exposed in the flanders schema and consequently in the graphql schema. In addition I would add the `search-txt` in the flanders schema. This field is deprecated like the `actions` field but can still be retrieved in old investigations.

Reply:
I add `actions`, but `search-text` contains a dash it cannot be used in graphql queries.